.ado/windows-vs-pr.yml

```diff
@@ -582,11 +582,11 @@ jobs:
       #    workingDirectory: current
       #  condition: and(succeeded(), or(eq(variables['BuildConfiguration'], 'DebugBundle'), eq(variables['BuildConfiguration'], 'ReleaseBundle')))
 
-  - template: templates/e2e-test-job.yml # Template reference
-    parameters:
-      name: E2ETest
-      BuildPlatform: x64
-      UseRNFork: true
+#  - template: templates/e2e-test-job.yml # Template reference
+#    parameters:
+#      name: E2ETest
+#      BuildPlatform: x64
+#      UseRNFork: true
```

Comment:
Thanks!  I filed this issue to track fixing and re-enable the tests:  [https://github.com/microsoft/react-native-windows/issues/3953](https://github.com/microsoft/react-native-windows/issues/3953)

Can you add a TODO here referencing the bug number?

Reply:
I don't think so. Your bug already linked with this PR

Reply:
I'm just asking for you to add a comment to the yml:
# TODO:  Temporarily disabled due to disk space issues.  See issue #3953